apps/guardian-ui/src/components/setup/screens/setConfiguration/SetConfiguration.tsx

```diff
@@ -260,40 +245,32 @@ export const SetConfiguration: React.FC<Props> = ({ next }: Props) => {
           isHost={isHost}
         />
       )}
+      <BasicSettingsForm
+        myName={myName}
+        setMyName={setMyName}
+        password={password}
+        setPassword={setPassword}
+        hasCopied={hasCopied}
+        onCopy={onCopy}
+        isFollower={!isHost && !isSolo}
+        hostServerUrl={hostServerUrl}
+        setHostServerUrl={setHostServerUrl}
+      />
       {error && (
         <Text color={theme.colors.red[500]} mt={4}>
           {error}
         </Text>
       )}
-      <Flex
-        direction='column'
-        justify='space-between'
-        alignSelf='center'
-        width={['100%', '100%', '60%']}
+      <Button
+        isDisabled={!isValid}
+        onClick={isValid ? handleNext : undefined}
+        leftIcon={<Icon as={ArrowRightIcon} />}
+        mt={['2', '4']}
+        width={['25%', 'auto']}
+        alignSelf='flex-start'
       >
-        {password !== '' && (
-          <Checkbox
-            isRequired
-            spacing='10px'
-            alignSelf='flex-start'
-            onChange={(e) => setPasswordCheck(e.target.checked)}
-          >
-            <Text color={theme.colors.yellow[500]}>
-              {t('set-config.admin-password-backup')}
-            </Text>
-          </Checkbox>
```

Comment:
Maybe instead of checking that checkbox we could require the user to surprise re-enter the password at the end? Or at least prompt that they need to back it up?

Reply:
I've got the confirmPassword to move to the next screen, I could add it to the verification step before they continue to the dashboard?